### PR TITLE
Update to polkadot-v1.1.0

### PR DESCRIPTION
**Pull request by weezy20**
Update to `Polkadot-Sdk v1.1.0`

### CODE REVIEW

**Comment by Moliholy** [comment]
@weezy20 cannot build with `--features runtime-benchmarks`.

**Review by Moliholy — CHANGES_REQUESTED** [review]

**Comment by Moliholy** [comment]
@weezy20 after my last commit it should build with `all-features` enabled and pass all the tests.

File: docs/zombienet.md [hunk 1]
```diff
@@ -41,9 +43,9 @@ Commands:
...
 
-⚠️ Take note: the path of the polkadot executable used there is `../bin/polkadot` which means you need to have a folder called `bin` outside the repository directory which contains your compiled polkadot binary which must be named as `polkadot`. Also please take care to compile the correct version of `polkadot` when testing out on your machine. You can do so by cloning `polkadot` and then checking out the appropriate branch that corresponds to the substrate/cumulus version used to build the template.
+⚠️ Note: The path of the polkadot executable used there is `./bin/polkadot` which means you need to have a folder called `bin` inside the repository directory which contains your compiled polkadot binary named as `polkadot`. In addition to that since version 1.1.0, you will also need to place the `polkadot-execute-worker` and the `polkadot-prepare-worker` binaries to be inside `bin`. As a user you don't need to interact with these but these are required to start up the main relay chain `polkadot` software. You can find all three [here](https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.1.0)
```
[the remark this answers is not in the document]
Comment (weezy20) [reply docs/zombienet.md:48]: I agree

File: node/src/service.rs [hunk 2]
```diff
@@ -520,50 +530,40 @@ fn build_consensus(
...
-		// We got around 500ms for proposing
-		block_proposal_slot_portion: SlotProportion::new(1f32 / 24f32),
-		// And a maximum of 750ms if slots are skipped
-		max_block_proposal_slot_portion: Some(SlotProportion::new(1f32 / 16f32)),
-		telemetry,
+		relay_chain_slot_duration,
+		proposer,
+		collator_service,
+		// Very limited proposal time.
+		authoring_duration: Duration::from_millis(500),
```
[the remark this answers is not in the document]
Comment (weezy20) [reply node/src/service.rs:557]: I believe this is the time that's spent for executing a parachain block which is 0.5 seconds, same as the previous block time. [Reference](https://github.com/paritytech/polkadot-sdk/blob/15a34838819f09f3bcfb3c05c08fb53934d8efc4/cumulus/parachain-template/node/src/service.rs#L413)

[the remark this answers is not in the document]
Comment (Moliholy) [reply node/src/service.rs:557]: Perfect then. Thanks for clarifying 👍

**Review by Moliholy — APPROVED** [review]

**Pull request merged**